<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>913</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>427</width>
      <height>751</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Hide information</string>
     </attribute>
     <widget class="QGroupBox" name="gbSelectionContainer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>400</width>
        <height>330</height>
       </rect>
      </property>
      <property name="title">
       <string>Container selection</string>
      </property>
      <widget class="QLabel" name="lblImgContainer">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>91</y>
         <width>381</width>
         <height>207</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Upload image</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblPathToContainer">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>21</y>
         <width>301</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>The path to the image in which you want to hide information:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblContainer">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>71</y>
         <width>241</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>The image in which you want to hide information:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbUploadImage">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>40</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lePathToContainer">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>41</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lblImgResolution">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>305</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Image resolution:</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gbInfoToHide">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>401</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Enter information to hide</string>
      </property>
      <widget class="QLabel" name="lblTextToHide">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Text to hide:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblCharLeft">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Characters left:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="teTextToHide">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>381</width>
         <height>131</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Get information</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>445</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
